tests/new_upgrades/test_hostgroup.py

```diff
@@ -4,7 +4,7 @@
 
 :CaseComponent: HostGroup
 
-:Team: Endeavour
+:Team: Proton
```

Comment:
@adamruzicka can you also please update code ownership of `tests/new_upgrades/test_hostgroup.py` to team Proton.